tests/test_clientsdaily.py

```diff
@@ -64,3 +64,61 @@ def test_to_profile_day_aggregates(spark):
     for k, expected in EXPECTED_INTEGER_VALUES.items():
         actual = int(result['sum({})'.format(k)])
         assert actual == expected
+
+
+def test_profile_creation_date_fields(spark):
+    from mozetl.clientsdaily import rollup
+
+    frame = make_frame_with_extracts(spark)
+    clients_daily = rollup.to_profile_day_aggregates(frame)
+
+    # Spark's from_unixtime() is apparently sensitive to environment TZ
+    # See https://issues.apache.org/jira/browse/SPARK-17971
+    expected_back = set([
+        u'2014-12-16', u'2016-09-07',
+        u'2016-05-12', u'2017-02-16',
+        u'2012-11-17', u'2013-09-08',
+        u'2017-02-12', u'2016-04-04',
+        u'2017-04-25', u'2015-06-17'
+    ])
+    expected_utc = set([
+        u'2014-12-17', u'2016-09-08',
+        u'2016-05-13', u'2017-02-17',
+        u'2012-11-18', u'2013-09-09',
+        u'2017-02-13', u'2016-04-05',
+        u'2017-04-26', u'2015-06-18'
+    ])
+    expected_forward = set([
+        u'2014-12-18', u'2016-09-09',
+        u'2016-05-14', u'2017-02-18',
+        u'2012-11-19', u'2013-09-10',
+        u'2017-02-14', u'2016-04-06',
+        u'2017-04-27', u'2015-06-19'
+    ])
+    ten_pcds = clients_daily.select("profile_creation_date").take(10)
```

Comment:
I'm not entirely sure what's happening here. Does the ordering matter here, or is it just taking 10 random values? Does this test also verify that profile creation dates are valid DateTime types?

Reply:
1. Per JIRA issue referenced in the comment a few lines up, Spark is sensitive to environment TZ but does not provide an API for controlling it. Therefore, any of three results can be correct.
2. Ordering doesn't matter.
3. The test doesn't validate that and, in fact, they aren't per the expression that creates actual1. Will fix in a followup PR.